### ISSUE TITLE
Videos in Spanish instead of English

### ISSUE THREAD

**Issue by Vistaus** [yuliskov/SmartTubeNext#2484]
I have every setting set to English (and I'm Dutch, so not from a Spanish speaking country), but since the last update, videos with both a English and Spanish dub will play in Spanish by default. Please fix this.

**Comment by ypk** [comment]
I came across something similar over the months. a handful of videos with english audio, starts playing in Russian. I've watched the same video on browser and on my mobile they play fine, but only in the app the audio is wrong.

So I think this isn't a language issue, but some sort of mixed streams? I'm not sure.

**Comment by Vistaus** [comment]
I hadn't experienced this issue until the update on May 7, when I opened this issue. And I don't experience it in web browsers (PC and mobile) either.

**Comment by bolabola6789** [comment]
YES!!! At least I'm not alone in this situation. Subtitles set in English, video language set in English in player settings and I still get Spanish dub when watching MKBHD's videos
